docs/chart-design-and-style/custom-themes-in-styled-mode.md

```diff
@@ -1,180 +1,64 @@
-    
-*   [sand-signika.css](https://code.highcharts.com/css/themes/sand-signika.css)
-    
-*   [grid-light.css](https://code.highcharts.com/css/themes/grid-light.css)
-    
-
-All these CSS files are generated from SCSS.
 
 How to create your own theme
 ----------------------------
 
-### A. Custom theme using CSS Rules
 
 Follow the steps below to create a custom theme using CSS rules:
 
 1.  Create a CSS file
-    
-2.  Use import to load all default highcharts styles
-    
-3.  Declare all classes which should be modified and adapt them by colors, fonts etc.
-    
-4.  Use the yourtheme.css file on a website.
-    
 
-Example of yourtheme.css
+2.  Use `@import` to load all default highcharts styles
+
+3.  Re-declare CSS variables of the default `highcharts.css` file. For more fine-grained control over each element's appearance, declare individual classes.
+
+
+Example of `yourtheme.css`
+
 
-    
     @import 'https://code.highcharts.com/css/highcharts.css';
-     
-    .highcharts-color-0 {
-      fill: #f45b5b;
-      stroke: #f45b5b;
+
+    /* This will re-declare variables */
+    :root {
+      --highcharts-color-0: #f45b5b;
+    }
+
+    /* This will use a different variable for some elements, in this case use a
+    more contrasted color variation for the axis labels */
+    .highcharts-axis-labels {
+      fill: var(--highcharts-neutral-color-100);
     }
-     
-    /* Titles */
+
+    /* Example of individual components overriding variables */
     .highcharts-title {
       fill: black;
       font-size: 26px;
       font-weight: bold;
     }
 
-### B. Custom theme using SASS
-
-Here are the steps to create a simple custom theme:
-
-1.  Create a SCSS file
-    
-2.  Declare variables which applies your styles
-    
-3.  Add additional classes which should be modified and adapt them by colors, fonts etc
-    
-4.  Use import to load all default highcharts styles
-    
-
-Example of yourtheme.scss
-
-    
-    // Colors for data series and points.
-    $colors: #7cb5ec #f7a35c #90ee7e #7798BF #aaeeee #ff0066 #eeaaee #55BF3B #DF5353 #7798BF #aaeeee;
-     
-    // Neutral colors
-    $neutral-color-100: #404048;
-    $neutral-color-80: #000;
-     
-    // Title
-    .highcharts-title, .highcharts-subtitle, .highcharts-yaxis .highcharts-axis-title {
-    text-transform: uppercase;
-    }
-     
-    @import 'highcharts';
-
-[View the live demo](https://codepen.io/Blacklabel/pen/wmZyLN)
-
-  
-  
-
-Remark
-
-Sass is an extension of CSS that allows you to use variables, nested rules and inline imports in stylesheet.
-
-In the [highcharts.scss](https://github.com/highcharts/highcharts/blob/master/css/highcharts.scss) file, there are declared variables which define the most important and popular elements of charts. Use them to easy style the chart in the customs theme. As a result, there is no need to overwrite all CSS classes.
-
-[View more about SASS](https://sass-lang.com/guide)
-
-Here are the list of all available variables:
```

Comment:
Shall we keep this legacy solution in docs? Eg under "vX-v10" paragraph or something similar?

Reply:
I added a Legacy section with a link to the [old docs article](https://github.com/highcharts/highcharts/blob/v10.3.3/docs/chart-design-and-style/custom-themes-in-styled-mode.md).